showcase/src/app/app.component.html

```diff
@@ -1,4 +1,131 @@
 <stark-app-container>
+	<header class="stark-app-header">
+		<div class="stark-header-container">
+			<div class="stark-app-bar">
+				<div class="stark-app-bar-content-left">
+					<div class="stark-actions">
+						<button class="stark-home-button" (click)="goHome()" color="white" mat-icon-button>
+							<mat-icon svgIcon="home"></mat-icon>
+						</button>
+						<button color="white" mat-icon-button (click)="toggleMenu()">
+							<mat-icon svgIcon="menu"></mat-icon>
+						</button>
+					</div>
+				</div>
+				<div class="stark-app-bar-content-center">
+					<div class="app-logo">
+						<stark-app-logo></stark-app-logo>
+					</div>
+				</div>
+				<div class="stark-app-bar-content-right">
+					<div>
+						<div *ngIf="!isMenuModeActive">
+							<stark-app-data mode="dropdown">
+								<div class="summary-slot">
+											<span *ngIf="user"
+												  class="label">{{ "SHOWCASE.DEMO.APP_DATA.USER" | translate }}</span
+											>: <span *ngIf="user" class="value">{{ user.username }} ({{ user.roles[0] }}
```

Comment:
Small issue here. Due to your change it is (there is a wrong space in front of the parenthesis): 
```
<span *ngIf="user" class="value">{{ user.username }} ({{ user.roles[0] }} )</span><br />
```

But it should be 
```
<span *ngIf="user" class="value">{{ user.username }} ({{ user.roles[0] }})</span><br />
```